File: src/org/nextras/orm/intellij/annotator/highlighter/ModifierHighlighter.java
package org.nextras.orm.intellij.annotator.highlighter;

import com.intellij.openapi.editor.DefaultLanguageHighlighterColors;
import com.intellij.openapi.editor.colors.TextAttributesKey;


public class ModifierHighlighter
{
	public static final TextAttributesKey BRACES = TextAttributesKey.createTextAttributesKey("BRACES", DefaultLanguageHighlighterColors.BRACKETS);
	public static final TextAttributesKey MODIFIER = TextAttributesKey.createTextAttributesKey("MODIFIER", DefaultLanguageHighlighterColors.KEYWORD);
	public static final TextAttributesKey IDENTIFIER = TextAttributesKey.createTextAttributesKey("IDENTIFIER", DefaultLanguageHighlighterColors.MARKUP_ATTRIBUTE);
}

File: src/main/kotlin/org/nextras/orm/intellij/annotator/highlighter/ModifierHighlighter.kt
package org.nextras.orm.intellij.annotator.highlighter

import com.intellij.openapi.editor.DefaultLanguageHighlighterColors
import com.intellij.openapi.editor.colors.TextAttributesKey

object ModifierHighlighter {
	val BRACES = TextAttributesKey.createTextAttributesKey("BRACES", DefaultLanguageHighlighterColors.BRACKETS)
	val MODIFIER = TextAttributesKey.createTextAttributesKey("MODIFIER", DefaultLanguageHighlighterColors.KEYWORD)
	val IDENTIFIER = TextAttributesKey.createTextAttributesKey("IDENTIFIER", DefaultLanguageHighlighterColors.MARKUP_ATTRIBUTE)
}
